src/DebugConfigurationProvider.ts

```diff
@@ -561,4 +567,10 @@ export interface BrightScriptLaunchConfiguration extends LaunchConfiguration {
      * If injectRdbOnDeviceComponent is true and this is true the screen saver will be be disabled while the deployed application is running.
      */
     disableScreenSaver?: boolean;
+
+    /**
+     * If true, the remote control will be enabled at the start of the debug session, and disabled at the end of the debug session.
+     * @default false
+     */
+    remoteControlMode?: boolean | { activateOnSessionStart?: boolean; deactivateOnSessionEnd?: boolean };
```

Comment:
Let's exclude the `boolean` part. The boolean shorthand can be converted to this in our DebugConfigurationProvider.